src/Nri/Ui/CharacterIcon/V1.elm

```diff
@@ -11,6 +12,69 @@ import Svg.Styled as Svg
FImayF3i0JVj5z2Tag2dLrmFVTBE4/ApBhvgm/ytHmSjQHBowZlBJAJpqbmYwk4F7PwH2Y8gAczCb5VwEZZC7LTtGSs6+Hmb38wOCYPNTQZ6SA9/vGDHFEMonRQcQ9DWRAMooBd45GSSWWYSDRRBJvFMEA0cuCut74BkmDJ0CGPtM+KI829ssuKAhsBaGdPE2gknxQe2MEfr630ygqEFcGWFjAjmQkCe1EkH4kwD/60Y9KdXX1G+9oBq/9P//yL3LPPfcYYvPtHKbL5TJZzHatd3sfJEVtYpf9AAlmbkPcglAJqK2tlYryckPok9x9I8sB+b9nzx5pgxQ8yV/ui3XkD/n8kFhPkBrUWe8EmU2ZbPY7H6pSdetj8fxdQ6+8hFr07BdolBvvHIUaAH0b14RZ9W63Wy5eu9bgyxIOJ2O8phdceKE8DnKeWfEk5tlf8Rq/HSPxycZ90W/oQwy0YQBIAEEkVAJgeYqqqqq3s/u3/R2W+SiHv3Yi2/+1114zx8X7n8oJHo/HqAB0dnaiLA4CAIAAj52Nzz720UQ9F/6ZCH/tRdZ3ENFda6HSos/Ft31Jzoov8tn9fJsPAX/jRgWAihD2+xWf/C6oA/DZw+XDUAngzRV9zkeDL+lj5VAZ8wdG5RMfvFEuXLlMMlEKwIH7uwflARrxdwaCprxQAqALMgjAjx3Uog3h3vrtWQGinoQioAgoAoqAIqAIKAKKgCKgCLxtBE7AXrztfekXFQFFQBFQBBSB2Y7AqziBL6MxK5rGNEMGAgSsWiilz58rkQ2bRHbs4bqjDYN01rVXYqQdj1YMaEcGfND2xAA5asXCODBHgpXyAMzKUVMEzigCSDzPg1MuwkFQ8j8Bg8XU+W7nQPQ4Bqw5AE2nPZGRUCLJxPIAhSgBQCPZpHY0Av8Xkv331XXL+o5Bk8F/P+bvfq1ZfrK7U/YjEADXwNgksCSBRFKNmLf4Q8CfJFl0PUkC/kdLBNvkAdZpjgRpxna8Dr7ghLQFQiD/hqVhMCh+ZKySJO0DCTWAdWqvjwDVE4rSnEaC2SZlSPZR+jsrM9NIpueBGJyt9tWvflXuvvtuU5f99c6BBNPrGdcxW9t3jOQ7l9NH7SlLA7AxWOCySy+V4qIiefyxx6SpqcmQzq+3f3t5a2uryQh//223yXnnnivXIhM7OzvbZMXuQWDLrw50m/uF2bKpzJhFH6V2NAJUCfGDQCShSElx4kZSMSUl2WSxtwFjBlpQUtzrPfnyIeci853Z8cyEZ9BAb28vYiKjFZKOPoKT/0Tfodn+Q2WJgw0NyHR2SQjZ9yTgz4Sloq56Y2OjZOLep/JHEfyYwQkk/rdv324w4HHx2ce+mOcxhsgt9t/4H2otIdnW65fh0LgUoF+h1LvamUUgAa83Z9IYxJeEVpiWbN6TvM5EKEVES2FwHQNE/qq6QIqhFMFnEEtIpGMbu3TEBP6eaO3ogvR/jWRnpJv7Y3DYb1QCWju7JTPdI3OKCvgeN4GevB3T53FjPYGoAM6/fud+JkHR31YEFAFFQBFQBBQBRUARUAQUgdOGwJn9i+i0nab+kCKgCCgCioAicNIIMK2tGI0pixwlZ7b+nyNtnW0omMy0N4k89hfI+wew+BjLB1FEMgXZjlYGvooBc4zScRnZUY4Ec79sHKxTUwTOHAKW1QBPLcEBkMEnm8YRaQ+JDTrqiWgLZjrao8mUqOV8AQat20F65CPzMZ4tFq8DQ2Oy6sGt8vO9XXLv9jb5xivN8ov9PfIsMvX3+UZBakaESgCxJCax5UUYRcAQpzbhz235H2sDk/QnOYDwDGkfCYPcj2aZchuWBGAgwNY+v2xoH5IuKAg8jazD/3cnOQC110PgwYO9IO3Ch7OkSUiGoPTCWvb76uqkHNnOJLZnozGz+r/uu89I97/R8fOcX89sojZ2vV3H3ZZz5zpmSpPEJYFKkripudlk8/eALD4Zu/zyy6Vu/37pg1Q9a7BzfyRf3SD7eXiUtg8go7oHGe1LUO5icFoq+2T2HS/bpIL4r85IkxD6A0rSsx9Bdy4eb4aRsH//Bz4g1193nSxbuvSkIbnxppuM/9gS/fSHkykd8WY/YPscr7PtTwz4aO/oMPfeqwgAePnll99sN6d8Pc+tAOoDK1asMIEOyXiXo3IC1S54rLE2gc8k/hkw50G/zMALErpezKciSOsgSgD0B9+8zEbsPnX+1CMQvRNO/X5Pdo8p8IkQ+i4G55xf4DWBACyd5Jjud1n2h8E6BXiHYhkJU7IDgSMMAuB3eROzHED/4BAqkHVKR3evtKMFRtHfoq/NzclCUI8J9Avjds/FPZppRSJb2KCkcnplNE4WFN1OEVAEFAFFQBFQBBQBRUARUAROGwKarnXaoNYfUgQUAUVAEZglCJDwZ1HNIjSOqrEswFbxj6wGse+2igslsnUnaPxCsYoKQPaTP40xlgmg3DGyJuVQM3Kr8ahNxkje+DgVAMi04EvyJ7R3lkKHHagpAm8XAUdSUh6+yzIADEbh+yADX7zIQHPaZPaxtKC9HNsZYyZudWaatEGam+R0OUoBxKsRKw7cf+PVZvmnra1Grp+1fknAjWE5M0FJNBsZf2xLEj8WT2b7ZaUkSaEr2Qz6D4UnTAagvRVVAkiakTMYCkWDBGys+V2WDmANYeqNNA6HZAyBBAehBtAwHMQ+nVKHcgMdo2H8hsOQVPZ343lKTP/PtlaDGedJblIGPA1ZyD6fz3wm5l/5u78zRMtsw+prX/uaPPfccyc8bJuA5coTkfyxX+K2kJg2i17ve1xO2X76IonS1rY2EwyQnZUll112WezuTjjPYIJ7//mfTdZ/CzLVO0AEt2EflJ/nvp0gwniNBqFsQZn1+SC6c6fVR064wzhbyL7kN/U9hngeBdEYYLkQ/JeE6+ZBtv+yZctkDTLaHXg3obw+AzXezD70oQ+ZzH+S4ry2VIKgUbr/nRr9iX7HRtWI9PR0Q7TzOPfu3WuuuxtKHBei9MDpNA
```

Comment:
Interesting that this is an encoded png! I take it there's not a straight svg without an encoded png in it available?